index.html

```diff
@@ -3,39 +3,43 @@
  <head>
    <title>The Gallery of Horns</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-   <link rel="stylesheet" href="layouts.css">
+   
+   <script id="creature-template" type="text/x-handlebars-template">
+    <section class="card" data-horns="{{horns}}" data-keyword="{{keyword}}">
+      <h2>{{title}}</h2>
+      <img
+        src="{{image_url}}"
+        alt="{{title}}">
+      <p>{{description}}</p>
+    </section>
+  </script>
+
+  <script id="creature-option-template" type="text/x-handlebars-template">
+    <option value="{{keyword}}">{{keyword}}</option>
+  </script>
+
+   <link rel="stylesheet" href="../css/layouts.css">
  </head>
  <body>
    <header>
-        <h1>The Gallery of Horns</h1>
-        <select id="horned_animals">
-          <option value="default">Filter by Keyword</option>
-          <option value="narwhal">Narwhal</option>
-          <option value="rhino">Rhino</option>
-          <option value="unicorn">Unicorn</option>
-          <option value="unilego">Unilego</option>
-          <option value="triceratops">Triceratops</option>
-          <option value="markhor">Markhor</option>
-          <option value="mouflon">Mouflon</option>
-          <option value="addax">Addax</option>
-          <option value="chameleon">Chameleon</option>
-          <option value="lizard">Lizard</option>
-          <option value="dragon">Dragon</option>
+    <h1>The Gallery of Horns</h1>
+      <select>
+        <option value="default">Filter by Keyword</option>
       </select>
-      </header>
-     </select>
-   </header>
+      <div id = "select_name">Select Creature Type</div>

```

Comment:
If this is a label for what the above selection box does, you can do it differently. Either by including some label in the select element, or following the notes I gave for line 27. 

Is this supposed to be the sort selection? If so, this should also be a select element and follow the pattern you had for your keyword filter. So between the start and close of the select element tags you can put the first option tag. That option tag can have a value="default" (or however you indicate the blank/original option), and inside the option element you tell the user to choose how to sort (or whatever this selection element is trying to do).